packages/ai-bot/main.ts

```diff
@@ -0,0 +1,303 @@
+import {
+  type IAuthData, MatrixEvent,
+  type RoomMember,
+  type EmittedEvents,
+  type IEvent,
+  type client,
+} from 'matrix-js-sdk';
+import * as MatrixSDK from 'matrix-js-sdk';
+// New
```

Comment:
Is this left over from something?

Reply:
Yeah that'll be left over from my dev - I'll get this file cleaned up.